muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/WorkspaceMaterialFieldAnswerController.java

```diff
@@ -116,6 +116,21 @@ public WorkspaceMaterialTextFieldAnswer updateWorkspaceMaterialTextFieldAnswerVa
     return workspaceMaterialTextFieldAnswerDAO.updateValue(workspaceMaterialTextFieldAnswer, value);
   }
   
+  /* JournalField */
```

Comment:
Are these three methods used at all? Doesn't WorkspaceJournalFieldIOHandler create, update, and find field data via WorkspaceJournalController?